go.mod

```diff
@@ -132,3 +134,15 @@ replace (
 	sigs.k8s.io/cluster-api-provider-azure => github.com/openshift/cluster-api-provider-azure v0.1.0-alpha.3.0.20200120114645-8a9592f1f87b // Pin OpenShift fork
 	sigs.k8s.io/cluster-api-provider-openstack => github.com/openshift/cluster-api-provider-openstack v0.0.0-20200526112135-319a35b2e38e // Pin OpenShift fork
 )
+
+replace (
+	// Pin the bellow modules not to be changed as result of github.com/hashicorp/terraform-provider-kubernetes addition
+	github.com/Azure/go-autorest/autorest/azure/cli => github.com/Azure/go-autorest/autorest/azure/cli v0.3.1
+	github.com/Azure/go-autorest/autorest/date => github.com/Azure/go-autorest/autorest/date v0.2.0
+	github.com/Azure/go-autorest/autorest/validation => github.com/Azure/go-autorest/autorest/validation v0.2.1-0.20191028180845-3492b2aff503
+	github.com/go-openapi/spec => github.com/go-openapi/spec v0.19.4
+	// Pin kubernetes hash
+	github.com/hashicorp/terraform-provider-kubernetes => github.com/hashicorp/terraform-provider-kubernetes v1.13.3-0.20201012211451-c24349628446
+	// Change kubevirt/terraform-provider-kubevirt to nirarg fork
```

Comment:
do we need the fork or not?

Reply:
Yes, we need it as long as the repository is not pushed to kubevirt
Pushing it to kubevirt organization work is in progress

Reply:
if there's a PR for that please add the link the comment